components/admin_console/billing/plan_details_jsx_pieces.tsx

```diff
@@ -0,0 +1,354 @@
+        localizeMessage('admin.billing.subscription.planDetails.features.guestAccounts', 'Guest Accounts'),
+        localizeMessage('admin.billing.subscription.planDetails.features.unlimitedIntegrations', 'Unlimited Integrations'),
+    ];
+
+    // const featuresCloudStarter = [
+    //     localizeMessage('admin.billing.subscription.planDetails.features.groupAndOneToOneMessaging', 'Group and one-to-one messaging, file sharing, and search'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.incidentCollaboration', 'Incident collaboration'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.unlimittedUsersAndMessagingHistory', 'Unlimited users & message history'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.unlimitedIntegrations', 'Unlimited Integrations'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.mfa', 'Multi-Factor Authentication (MFA)'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.multilanguage', 'Multi-language translations'),
+    // ];
+
+    const featuresCloudProfessional = [
+        localizeMessage('admin.billing.subscription.planDetails.features.advanceTeamPermission', 'Advanced team permissions'),
+        localizeMessage('admin.billing.subscription.planDetails.features.mfaEnforcement', 'MFA enforcement'),
+        localizeMessage('admin.billing.subscription.planDetails.features.multiplatformSso', 'Gitlab, Google, and O365 single sign-on'),
+        localizeMessage('admin.billing.subscription.planDetails.features.guestAccounts', 'Guest Accounts'),
+        localizeMessage('admin.billing.subscription.planDetails.features.channelModeration', 'Channel moderation'),
+        localizeMessage('admin.billing.subscription.planDetails.features.readOnlyChannels', 'Read-only announcement channels'),
+    ];
+
+    // const featuresCloudEnterprise = [
+    //     localizeMessage('admin.billing.subscription.planDetails.features.enterpriseAdministration', 'Enterprise administration & SSO'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.autoComplianceExports', 'Automated compliance exports'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.customRetentionPolicies', 'Custom data retention policies'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.sharedChannels', 'Shared channels (coming soon)'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.enterpriseAdminSso', 'Enterprise administration & SSO'),
+    //     localizeMessage('admin.billing.subscription.planDetails.features.premiumSupport', 'Premium Support (optional upgrade)'),
+    // ];
+
+    let features = featuresFreeTier;
+
+    if (isPaidTier) {
+        switch (typeSubscription) {
+        case 'FREE_TRIAL':
```

Comment:
We shouldn't mix trial with type of subscription, same comment as above to always show `Cloud Professional` now or the free tier case below